srcpkgs/libmt32emu/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'libmt32emu'
+pkgname=libmt32emu
+version=2.5.1
+revision=1
+_version_string=${version//./_}
+wrksrc=munt-libmt32emu_${_version_string}
+build_wrksrc=mt32emu
+build_style=cmake
+makedepends="$(vopt_if libsoxr libsoxr-devel)
+ $(vopt_if libsamplerate libsamplerate-devel)"
+short_desc="Library to emulate Roland MT-32, CM-32L and LAPC-I synthesizer
+ modules"
+maintainer="Joshua Krämer <joshua@kraemer.link>"
+license="LGPL-2.1-or-later"
+homepage="http://munt.sourceforge.net"
+distfiles="https://github.com/munt/munt/archive/libmt32emu_${_version_string}.tar.gz"
+checksum=684988ab4d2b103554d9a38ad556cebc3ebc28d1d8fbb49fb4808ec9c80a2bf4
+
+build_options="libsoxr libsamplerate"
+desc_option_libsoxr="Use libsoxr for sample rate conversion"
+desc_option_libsamplerate="Use libsamplerate for sample rate conversion"
+build_options_default="libsoxr"
+vopt_conflict libsoxr libsamplerate
+
+if [ "$build_option_libsoxr" -o "$build_option_libsamplerate" ]; then
+	configure_args+=" -Dlibmt32emu_WITH_INTERNAL_RESAMPLER=off"
+fi
+
+post_install() {
+	rm -rf ${DESTDIR}/usr/share/doc
```

Comment:
Use `-r` so we get an error if they stop installing the files.

Reply:
Done